code/src/components/Album.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import Artists from './Artists';
+import IconsOverlay from './IconsOverlay';
+
+const Album = (props) => {
+  const { src, title, artistsArray, albumURL } = props;
+  return (
+    <article className='song-card'>
+      <div className='album-art-container'>
+        <img className='song-image' src={src} alt='album cover' />
+        <IconsOverlay />
+      </div>
+      <a href={albumURL}>
```

Comment:
Could be nice to add target="blank" to get the song in a new tab and not leave the site. Also you could add rel="noreferrer noopener" for security reasons (just learned from Lousanne!). Read more here: https://www.reliablesoft.net/noreferrer-noopener/